smac/facade/smac_ac_facade.py

```diff
@@ -427,6 +427,9 @@ def __init__(self,
             'min_chall': scenario.intens_min_chall  # type: ignore[attr-defined] # noqa F821
         }
 
+        if intensifier is None:
+            intensifier = Intensifier
+
         if isinstance(intensifier, Intensifier) \
```

Comment:
if we would allow all these (None, intensifier class, intensifier instance), we should update the documentation to make it clear what to pass to __init__

Reply:
I have updated the documentation.